simtools/configuration/commandline_parser.py

```diff
@@ -46,6 +47,8 @@ def initialize_default_arguments(
             Add output file configuration to list of args.
         telescope_model: bool
             Add telescope model configuration to list of args.
+        site_model: bool
```

Comment:
Just a comment (no action needed): I tend to think that flags have to come with a verb in their names, otherwise I expect a model from an argument called `site_model` but I see that all the other parameters here also don't have verbs so I won't suggest changing everywhere.

Reply:
Yes, here I followed all the other parameters to be consistent.